test/bin/find.js

```diff
@@ -108,4 +108,35 @@ describe('bin', function() {
     proxyquire('../../src/bin/find', stub)
     assert.ok(getCurrentRules.called)
   })
+
+  it('logs core rules', function() {
+    var stub = {
+      '../lib/rule-finder': function(specifiedFile, noCore) {
+        return {
+          getCurrentRules: function() {
+            assert(!noCore)
+            return ['current', 'rules']
+          }
+        }
+      },
+    }
+    process.argv[2] = '-c'
+    proxyquire('../../src/bin/find', stub)
+  })
+
+  it('does not log core rules', function() {
+    var stub = {
+      '../lib/rule-finder': function(specifiedFile, noCore) {
+        return {
+          getCurrentRules: function() {
+            assert(noCore)
+            return ['current', 'rules']
+          }
+        }
+      },
+    }
+    process.argv[2] = '-c'
```

Comment:
either of `c`, `a`, `u` or ` p`  is required.
https://github.com/sarbbottam/eslint-find-rules/blob/master/src/bin/find.js#L34-L39

```
if (!argv.c && !argv.p && !argv.a && !argv.u) {
  console.log('no option provided, please provide a valid option') // eslint-disable-line no-console
  console.log('usage:') // eslint-disable-line no-console
  console.log('eslint-find-rules [option] <file> [flag]') // eslint-disable-line no-console
  process.exit(0)
}
```

>test should be added that only does `--no-core`

`--no-core` w/o an option would error out.